.env.production

```
@@ -1 +1,2 @@
-REACT_APP_API=https://freefrom-compensation-api.herokuapp.com
\ No newline at end of file
+REACT_APP_API=https://freefrom-compensation-api.herokuapp.com
```

I think the API URL has changed
https://ragtag-team.slack.com/archives/CJ79S923E/p1574096217204100
```suggestion
REACT_APP_API=https://compensation-api.freefrom.org/
```